### PR TITLE
Fixed Md formatting

### PR DESCRIPTION
**Pull request by NicholasNeshev**
Fixed README file.

### CODE REVIEW

**Review by elnesh — COMMENTED** [review]
Loock good :)

File: TortoiseAndHare/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-The Tortoise moves steadily, taking smaller steps, but rarely stepping backwards.
-The Hare moves sporadically, leaping larger distances, often times backwards.
-The "OUCH!!" message is displayed when the two move to the same space.
-The winner is diplayed when they reach the end of the track.
+## TortoiseAndHare
+* Simulates the classic Tortioise and Hare race.
+* The Tortoise moves steadily, taking smaller steps, but rarely stepping backwards.
+* The Hare moves sporadically, leaping larger distances, often times backwards.
+* The "OUCH!!" message is displayed when the two move to the same space.
+* The winner is diplayed when they reach the end of the track.
```
Comment (nashneshev) [inline TortoiseAndHare/README.md:6]: Consider also adding a paragraph to explain the Java features demonstrated in this project.

**Review by elnesh — COMMENTED** [review]
Looks good :)

**Review by nashneshev — APPROVED** [review]

**Review by elnesh — APPROVED** [review]
Looks good :)

**Pull request merged**